src/ECharts/Chart.purs

```diff
@@ -0,0 +1,106 @@
+module ECharts.Chart where
```

Comment:
The modules should only export the public, "clean" API. Currently, the modules export everything, even the dirty / unsafe / private functions not intended to be used by users.